core/system/src/mpu/vmpu_freescale_k64_mem.c

```diff
@@ -269,7 +269,7 @@ void vmpu_mem_init(void)
 
     /* rest of SRAM, accessible to mbed - non-executable for uvisor */
     res = vmpu_mem_add_int(0, __uvisor_config.bss_end,
-        ((uint32_t) __uvisor_config.sram_end) - ((uint32_t) __uvisor_config.bss_end),
+        UVISOR_REGION_ROUND_UP((uint32_t) __uvisor_config.page_start) - ((uint32_t) __uvisor_config.bss_end),
```

Comment:
`page_allocator_init` rounds this up as well, so the values should be the same.
Unfortunately I cannot use `g_page_heap_start` at this point, since `vmpu_init` is called before `page_allocator_init`.